R/render.r

```diff
@@ -27,10 +27,8 @@ render_page <- function(pkg = ".", name, data, path = "", depth = NULL, quiet =
     depth <- length(strsplit(path, "/")[[1]]) - 1L
   }
 
-  data$logo <- list(
-    src = logo_path(pkg, depth = depth),
-    id = if (is.null(data$source)) "logo-low" else "logo-high"
-  )
+  data$logo <- list(src = logo_path(pkg, depth = depth))
+  data$logo$headerclass <- if (is.null(data$logo$src)) "" else "with-logo"
```

Comment:
I've realized I needed to use another condition... Depending on the presence of a logo.

Reply:
Hmmm, are you sure it's necessary? I think it's fine to leave the space for the logo even if it isn't present. It's certainly much simpler.

Reply:
Ok, done. :-)